CHANGELOG.md

```diff
@@ -2,18 +2,36 @@
 
 ## Unreleased
 
+## v0.6.0 Beta
+
 ## 🛑 Breaking changes 🛑
 
 - Renamed the metrics generated by `hostmetrics` receiver to match the (currently still pending) OpenTelemetry system metric conventions (#1261) (#1269)
-
-## 🚀 New components 🚀
+- Removed `vmmetrics` receiver (#1282)
+- Removed `cpu` scraper `report_per_cpu` config option (#1326)
 
 ## 💡 Enhancements 💡
 
 - Added disk merged (#1267) and process count (#1268) metrics to `hostmetrics`
+- Log metric data points in `logging` exporter (#1258)
+- Changed the `batch` processor to not ignore the errors returned by the exporters (#1259)
+- Build and publish MSI (#1153) and DEB/RPM packages (#1278, #1335)
+- Added batch size metric to `batch` processor (#1241)
+- Added log support for `memorylimiter` processor (#1291) and `logging` exporter (#1298)
+- Always add tags for `observability`, other metrics may use them (#1312)
+- Added metrics support (#1313) and allow partial retries in `queued_retry` processor (#1297)
+- Update `resource` processor: introduce `attributes` config parameter to specify actions on attributes similar to `attributes` processor, old config interface is deprecated (#1315)
+- Update memory state labels for non-Linux OSs (#1325)
+- Ensure tcp connection value is provided for all states, even when count is 0 (#1329)
+- Set `batch` processor channel size to num cpus (#1330)
 
 ## 🧰 Bug fixes 🧰
 
+- Fixed OTLP waitForReady, not set from config (#1254)
+- Fixed all translation diffs between OTLP and Jaeger (#1222)
+- Disabled `process` scraper for any non Linux/Windows OS (#1328)
+- Fix overflowing batch size in `batch` processor (#1310)
```

Comment:
This is not a bugfix actually. Could you move it to `Enhancements`:

- Add `send_batch_max_size ` config parameter to `batch` processor enforcing hard limit on batch size